java/src/jmri/jmrit/display/CoordinateEdit.java

```diff
@@ -332,7 +332,7 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
     public void initSetLevel() {
         oldX = pl.getDisplayLevel();
         textX = new javax.swing.JLabel();
-        textX.setText("level= " + pl.getDisplayLevel());
+        textX.setText(Bundle.getMessage("Level") + " = " + pl.getDisplayLevel());
```

Comment:
As you are working on the initial translation of these, you want to go ahead and include the variables in the string in the .properties file.